package.json

```diff
@@ -71,7 +71,7 @@
     "@cerner/duplicate-package-checker-webpack-plugin": "^1.0.0",
     "async": "^3.0.1",
     "autoprefixer": "^9.5.1",
-    "axe-core": "3.2.2",
+    "axe-core": "3.4.0",
```

Comment:
Can we open up the range ?

Reply:
Based on the this update being non passive, it seems like we should not open the range and evaluate each release.